blog/models.py

```diff
@@ -78,6 +79,11 @@ def get_context(self, request, tag=None, category=None, author=None, *args,
             else:
                 blogs = blogs.filter(author_id=author)
 
+        search = ''
+        if request.GET.get('search'):
```

Comment:
Please add an explanation comment here. It's not clear to me why search would be in here - is this a default wagtail thing?